### PR TITLE
feat(#24): complete the about us page for first time

### LINKED ISSUE #24: feat: Update About Us and other page content
** discord username: `@Mustafa Bolat #4560` **

### PR DESCRIPTION
**Pull request by salihdev0**
# Description

Hakkımızda sayfası geliştirildi. Kullanım kılavuzu gibi kullanılıyor.

Text verileri henüz hazır olmadığı için pr'ı bekletmeyi tavsiye ediyorum.

**discord username: ssi#9999**

**closes #24**

## Things to check before creating a PR
- [ ] I have inspected my topic, checked.
- [x] If it is a core feature, executed detailed tests.

## Creating PR rules
- [x] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [x] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [x] Related file selection: Only relevant files should be touched and no other files should be affected.
- [x] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [x] Clean commit history: The commits where the changes are made should be clear and organized.
- [x] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [x] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

### Mobil Görünüm

Screenshot 2023-05-23 232931

### Web Görünüm 

Screenshot 2023-05-23 232554

### CODE REVIEW

File: src/components/AboutUs/AboutUsOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  const aboutView = useAboutView();
+  if (!aboutView.isOpen) return null;
+  return (
+    <Fade in={aboutView.isOpen}>
+      <Container sx={styles.container}>
+        <PopupHeader
+          onBack={() => {
+            aboutView.toggle(!aboutView.isOpen);
+          }}
+        ></PopupHeader>
```
Comment (usirin) [inline src/components/AboutUs/AboutUsOverlay.tsx]: Let's change this to a self closing tag.

```suggestion
        />
```

File: src/components/AboutUs/AboutUsOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    [theme.breakpoints.up("lg")]: {
+      maxWidth: 550,
+      height: "auto",
+      borderRadius: "8px !important",
+    },
+    [theme.breakpoints.up("xl")]: {
+      maxWidth: 550,
+      height: "auto",
+      borderRadius: "8px !important",
```
Comment (usirin) [inline src/components/AboutUs/AboutUsOverlay.tsx]: why do we need `!important` for these? 

If it's about trying to make a pixel-perfect match for the figma file, let's not do that. The figma designs are there as a reference and we ideally only want to use our default styles.

**Review by usirin — CHANGES_REQUESTED** [review]
Thank you so much! Great first contribution! 💯

I left couple of comments, let me know when you address them!

File: src/components/AboutUs/AboutUsOverlay.tsx (hunk 2 above)
Comment (salihdev0) [reply src/components/AboutUs/AboutUsOverlay.tsx]: yes it was for pixel perfect. So I remove

**Review by usirin — APPROVED** [review]
💯

**Pull request merged**